JPA testProducerNativeQuery fails for MySQL and MariaDB databases

### Bug description

The native query to run is:

```
SELECT * FROM fruit WHERE id = :id
```

In the dev services containers, `lower_case_table_names` is probably set to the default (`0`). So the table name will be created and stored as `Fruit`. Thus the query will fail like:

```
Caused by: java.sql.SQLSyntaxErrorException: (conn=4) Table 'quarkus.fruit' doesn't exist
```

H2 is likely more forgiving about entity case sensitivity, hence why this issue has gone unnoticed.

--- a/integration-tests/jpa/src/main/java/org/apache/camel/quarkus/component/jpa/it/JpaRoute.java
+++ b/integration-tests/jpa/src/main/java/org/apache/camel/quarkus/component/jpa/it/JpaRoute.java
@@ -48,7 +48,7 @@ public class JpaRoute extends RouteBuilder {
                 .to(jpaEndpoint + "?namedQuery=findByName&parameters=#parameters");
         from("direct:nativeQuery")
                 .to(jpaEndpoint + "?resultClass=org.apache.camel.quarkus.component.jpa.it.model.Fruit" +
-                        "&nativeQuery=SELECT * FROM fruit WHERE id = :id");
+                        "&nativeQuery=SELECT * FROM Fruit WHERE id = :id");
         from("direct:store")
                 .to(jpaEndpoint);
         from("direct:remove")

